release-notes/10.0/preview/preview3/wpf.md

```diff
@@ -0,0 +1,13 @@
+# WPF in .NET 10 Preview 3 - Release Notes
+
+Here's a summary of what's new in WPF in this preview release:
+
+- [Feature](#feature)
+
+WPF updates in .NET 10:
+
+- [What's new in WPF in .NET 10](https://learn.microsoft.com/dotnet/desktop/wpf/whats-new/net100) documentation.
+
+## Feature
+
+Something about the feature

```

Comment:
are you sure you want to merge this as is?

Reply:
Merge this first, into this branch: #9844